### PR TITLE
Working with Gemini notebook

### PR DESCRIPTION
**Pull request by anushas**

### CODE REVIEW

**Comment by aaron-siegel** [comment]
This change is [Reviewable](https://reviewable.io/reviews/pixeltable/pixeltable/393)

File: docs/notebooks/integrations/working-with-gemini.ipynb [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from pixeltable.functions import gemini\n",
+    "\n",
+    "# Create a table in Pixeltable and pick a model hosted on Googel AI Studio with some parameters\n",
```
Comment (pierrebrunelle) [inline docs/notebooks/integrations/working-with-gemini.ipynb]: Typo: **Googel AI**

Comment (anushas) [reply docs/notebooks/integrations/working-with-gemini.ipynb]: fixed.

**Review by aaron-siegel — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pixeltable/pixeltable/393)* status: 0 of 1 files reviewed, 3 unresolved discussions (waiting on @anushas and @pierrebrunelle)

___
*[`docs/notebooks/integrations/working-with-gemini.ipynb` line 170 at r3](https://reviewable.io/reviews/pixeltable/pixeltable/393#-ODIb2Y7Dn0AXv0t6qxu:-ODIb2Y7Dn0AXv0t6qxv:b-s9g6cx) ([raw file](https://github.com/pixeltable/pixeltable/blob/0a2d86aa7a6fe7e2feef1295d65b67d6de601307/docs/notebooks/integrations/working-with-gemini.ipynb#L170)):*
> ```
>     "            stop_sequences=['\\n'],\n",
>     "            max_output_tokens=300,\n",
>     "            temperature=1.0,\n",
> ```

I don't think we need so many of these extra arguments. A typical application wouldn't use them all - maybe just include a few as in the Mistral and Anthropic notebooks etc? The full range of options are in the API docs if users need them.

Also, let's strive to follow proper Python style in indentation (4 spaces at a time)
___
*[`docs/notebooks/integrations/working-with-gemini.ipynb` line 213 at r3](https://reviewable.io/reviews/pixeltable/pixeltable/393#-ODIbbyHBGCuD4jyzxbI:-ODIbbyHBGCuD4jyzxbJ:b-8j3bk0) ([raw file](https://github.com/pixeltable/pixeltable/blob/0a2d86aa7a6fe7e2feef1295d65b67d6de601307/docs/notebooks/integrations/working-with-gemini.ipynb#L213)):*
> ```
>    "source": [
>     "# Ask Gemini to generate some content based on the input\n",
>     "t.insert([{'input': 'Write a story about a magic backpack.'}, \n",
> ```

Follow proper Python style here for a multi-line list:

```python
t.insert([
    {'input': 'Write a story about a magic backpack.'},
    {'input': 'Tell me a science joke.'}
])
```

**Review by anushas — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pixeltable/pixeltable/393)* status: 0 of 1 files reviewed, 3 unresolved discussions (waiting on @aaron-siegel and @pierrebrunelle)

___
*[`docs/notebooks/integrations/working-with-gemini.ipynb` line 170 at r3](https://reviewable.io/reviews/pixeltable/pixeltable/393#-ODIb2Y7Dn0AXv0t6qxu:-ODN7LUR8kjWrxcE1BZE:b-896fix) ([raw file](https://github.com/pixeltable/pixeltable/blob/0a2d86aa7a6fe7e2feef1295d65b67d6de601307/docs/notebooks/integrations/working-with-gemini.ipynb#L170)):*
<details><summary><i>Previously, aaron-siegel (Aaron Siegel) wrote…</i></summary><blockquote>

I don't think we need so many of these extra arguments. A typical application wouldn't use them all - maybe just include a few as in the Mistral and Anthropic notebooks etc? The full range of options are in the API docs if users need them.

Also, let's strive to follow proper Python style in indentation (4 spaces at a time)
</blockquote></details>

Done.
___
*[`docs/notebooks/integrations/working-with-gemini.ipynb` line 213 at r3](https://reviewable.io/reviews/pixeltable/pixeltable/393#-ODIbbyHBGCuD4jyzxbI:-ODN7MDw-op8NpfrktQq:b-896fix) ([raw file](https://github.com/pixeltable/pixeltable/blob/0a2d86aa7a6fe7e2feef1295d65b67d6de601307/docs/notebooks/integrations/working-with-gemini.ipynb#L213)):*
<details><summary><i>Previously, aaron-siegel (Aaron Siegel) wrote…</i></summary><blockquote>

Follow proper Python style here for a multi-line list:

```python
t.insert([
    {'input': 'Write a story about a magic backpack.'},
    {'input': 'Tell me a science joke.'}
])
```
</blockquote></details>

Done.

**Review by pierrebrunelle — APPROVED** [review]
LGTM

**Review by aaron-siegel — APPROVED** [review]

**Pull request merged**